lib/fastly_nsq/listener.rb

```diff
@@ -4,10 +4,11 @@ def self.listen_to(**args)
       new(**args).go
     end
 
-    def initialize(topic:, processor:, channel: nil, consumer: nil)
-      @topic     = topic
-      @processor = processor
-      @consumer  = consumer || FastlyNsq::Consumer.new(topic: topic, channel: channel)
+    def initialize(topic:, processor:, channel: nil, consumer: nil, preprocessing: nil)
+      @topic         = topic
+      @preprocessing = preprocessing
```

Comment:
Would be nice to see a note about the preprocessor in the README or the examples